app/controllers/comments_controller.rb

```diff
@@ -75,9 +75,10 @@ def create
 
       if @comment.invalid?
         @comment.destroy
-        render json: { status: "comment already exists" }
+        render json: { error: "comment already exists" }, status: :conflict
```

Comment:
Interesting choice. I usually use 422 in these cases, since 409 [says](https://httpstatuses.com/409) 

> This code is used in situations where the user might be able to resolve the conflict and resubmit the request.

Reply:
I agree with you! I was conflicted (LOL) in which one to use but I didn't check the specs